apps/src/StudioApp.js

```diff
@@ -2855,6 +2865,19 @@ StudioApp.prototype.polishGeneratedCodeString = function (code) {
 };
 
 /**
+ * Returns whether this view should be responsive based on level options.
+ * Responsive means that the visualizationColumn should resize as the
+ * window width changes, and that a grippy is available to manually resize
+ * the visualizationColumn.
+ */
+StudioApp.prototype.isResponsiveFromConfig = function (config) {
+  // use config.responsiveEmbedded because this.responsiveEmbedded may not be initialized yet
```

Comment:
I kept this for documentation purposes, and so that anyone needing to refer to this would use `this.responsiveEmbedded` instead of `config.responsiveEmbedded`, as is done with `wireframeShare` and `centerEmbedded`. However I'm happy to remove `this.responsiveEmbedded` if you prefer.